blocks.py

```diff
@@ -1,58 +1,54 @@
+from nio import GeneratorBlock
+from nio.properties.version import VersionProperty
+
 from .multiple import MultipleSignals
 from .generators.counter import CounterGenerator
 from .generators.identity import IdentityGenerator
 from .generators.file import FileGenerator
 from .triggers.interval import IntervalTrigger
 from .triggers.safe import SafeTrigger
 from .triggers.cron import CronTrigger
-from nio import Block, discoverable
-from nio.properties.version import VersionProperty
 
 
-@discoverable
 class CounterIntervalSimulator(
         MultipleSignals,
         CounterGenerator,
         IntervalTrigger,
-        Block):
+        GeneratorBlock):
```

Comment:
While we do want to use these, they don't work in nio until the next release so we should stay away from it for now.